central/auth/datastore/datastore_impl.go

```diff
@@ -154,7 +154,7 @@ func (d *datastoreImpl) InitializeTokenExchangers() error {
 		// This is required for config-controller auth.
 		configs = append(configs, &storage.AuthMachineToMachineConfig{
 			Type:                    storage.AuthMachineToMachineConfig_KUBE_SERVICE_ACCOUNT,
-			TokenExpirationDuration: "1h",
+			TokenExpirationDuration: "1m",
```

Comment:
revert?

Reply:
I admit this change is not related to CRD, so perhaps I can open it as a separate PR.  This is also in #12630

Reply:
Oh ok, so it was intentional. Got it.